package.json

```diff
@@ -1,16 +1,12 @@
 {
-  "name": "js-to-fhirpath",
-  "version": "1.1.0",
+  "name": "easy-path-expressions",
+  "version": "2.0.0",
   "type": "module",
-  "description": "js to fhirpath syntax converter",
+  "description": "JS to FHIRPath syntax converter",
```

Comment:
In the README, you say it "converts basic mathematical expressions into FHIRPath", which I think is more accurate.

Reply:
Updated.